src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -382,11 +539,8 @@ public bool Disconnect(DisconnectParams disconnectParams)
                 }
             }
 
-            // Close the connection            
-            info.SqlConnection.Close();
-
-            // Remove URI mapping
-            ownerToConnectionMap.Remove(disconnectParams.OwnerUri);
+            // TODO do these activities depend on the type of connection?
```

Comment:
I'm thinking only on the default connection. We run the risk of duplicating actions (that may or may not be idempotent) if we run this for every connection. Not only that but extra actions may slow down the service.